Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gcc:9.2
```

Comment:
Can we do Ubuntu too?

Reply:
Yes, we can.

But [GCC](https://hub.docker.com/_/gcc) image is based on Debian, do you think it's necessary?